.github/workflows/sauce.yml

```diff
@@ -0,0 +1,27 @@
+name: Sauce Labs
+on: push
+This conversation was marked as resolved by marcuslyons
+ Show conversation
+concurrency: sauce-labs
+jobs:
+  test:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v1
+      - name: Set up node
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14
+      - name: Install
+        run: npm install
+        env:
+          # Download Sauce Connect binary now instead of on first run
+          SAUCE_CONNECT_DOWNLOAD_ON_INSTALL: true
+      - name: Add host
+        run: echo "127.0.0.1 airtap.local" | sudo tee -a /etc/hosts
+      - name: Test
+        run: npm run test-browsers
+        env:
+          SAUCE_USERNAME: ${{ secrets.SAUCE_USERNAME }}
+          SAUCE_ACCESS_KEY: ${{ secrets.SAUCE_ACCESS_KEY }}
```

Comment:
We also have coverage data here so let's upload it, by adding the following steps at the end:

```yaml
- name: Coverage
  run: npm run coverage
- name: Codecov
  uses: codecov/codecov-action@v1
  with:
    file: coverage/lcov.info
```

Reply:
Added in 96172d1